projects/fourier/lib/fourier/utilities/swift_package_manager.rb

```diff
@@ -7,31 +7,67 @@ module SwiftPackageManager
       ARM64_TARGET = "arm64-apple-macosx"
       X86_64_TARGET = "x86_64-apple-macosx"
 
-      def self.build_fat_release_library(path:, product:, binary_name:, output_directory:, swift_build_directory:)
-        FileUtils.mkdir_p(File.expand_path("#{product}.swiftinterface", output_directory))
-        FileUtils.mkdir_p(File.expand_path("#{product}.swiftmodule", output_directory))
-
-        self.build_fat_release_binary(
-          path: path,
-          product: product,
-          binary_name: binary_name,
-          output_directory: output_directory,
-          swift_build_directory: swift_build_directory
-        ) do |arch|
-          additional_options = [
+      def self.build_fat_release_library(path:, product:, output_directory:, swift_build_directory:)
+        release_directory = File.expand_path("apple/Products/Release/", swift_build_directory)
+
+        unless File.exist?(release_directory)
+          FileUtils.mkdir_p(release_directory)
+        end
+
+        build = -> (arch) {
+          Utilities::System.system(
+            "swift", "build",
+            "--configuration", "release",
+            "--disable-sandbox",
+            "--package-path", path,
+            "--product", product,
+            "--build-path", swift_build_directory,
+            "--triple", "#{arch}-apple-macosx",
             "-Xswiftc", "-enable-library-evolution",
```

Comment:
I know this isn't part of the diff but why does Tuist need to enable library evolution at all?